nix/tullia.nix

```diff
@@ -0,0 +1,123 @@
+self: system:
```

Comment:
Can we add some comment here
- what the purpose/intention of this file is?
- what `tullia` is? 
- where to find the corresponding documentation or this file?

I've no read this file a few times, but I still don't think I fully understand why it is I this repo?
There seem to be 20 repo relevant lines from L102 to L122 the rest is ceremony?

Reply:
> I've no read this file a few times, but I still don't think I fully understand why it is I this repo?
> There seem to be 20 repo relevant lines from L102 to L122 the rest is ceremony?

As `hydraJobs` in this flake does not follow the flake output schema of `hydraJobs."<attr>"."<system>" = derivation;`, we have to write code specific to this repo in order to generate Tullia tasks from it that can not be stored away in some tullia library.
If this flake is converted to follow the flake output schema later we can likely make this file a bit smaller by writing a generalized hydra jobs to Tullia tasks function in Tullia's library.

Reply:
> There's not much else to say if the reader knows what Tullia and Cicero are

Please assume the reader knows nothing.

Reply:
@dermetfan Yes, we don't document `hydraJobs` either. That's not a good precedent. That's actually an example of where we fall massively short.

I think this file should say that the file contains Tulia tasks and Cicero actions. And also provide links as to where one can read more about them. We want to eventually turn this into an open source project, where others need to be able to find out what's going on without having access to IOG slack or other IOG folks readily. It needs to be sufficiently self explanatory.

Reply:
Just to stress this again. It doesn't need to contain the full writeup what Tulia and Cicero are. But it needs to have sufficient links to where one can read up on it to make sense of this file without having to informally consult someone else.

Reply:
I added a short description to the top of this file. Please let me know whether you think that is sufficient.